### ISSUE TITLE
extract_hobo.py does not insert if csv has a mix of old and new readings

### ISSUE THREAD

**Issue by matthew-schultz** [erdl/sensors#89]
If a hobo csv has readings already in the database __AND__ readings that are not in the database, extract_hobo.py will not insert the new readings.

**Comment by carlosparadis** [comment]
I'm assuming this issue is a reminder that this should be added to the documentation?

**Comment by matthew-schultz** [comment]
Yeah, unless we wanted to try to update extract_hobo.py to deal with cases like that.

**Comment by matthew-schultz** [comment]
update extract_hobo to use if statement to check if < last_updated_datetime before inserting
